ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Mock/Block/PBFT.hs

```diff
@@ -91,11 +94,18 @@ instance ( SimpleCrypto c
       , headerSigned hdr
       )
 
+instance (PBftCrypto c', Serialise (PBftVerKeyHash c'))
+      => RunMockProtocol (PBft ext c') where
+  mockProtocolMagicId  = const constructMockProtocolMagicId
+  mockEncodeChainState = const CS.encodePBftChainState
+  mockDecodeChainState = const CS.decodePBftChainState
+
 instance ( SimpleCrypto c
          , PBftCrypto c'
          , Signable (PBftDSIGN c') (SignedSimplePBft c c')
          , ConstructContextDSIGN ext c'
-         ) => ForgeExt (PBft ext c') c (SimplePBftExt c c') where
+         , Serialise (PBftVerKeyHash c')
```

Comment:
Why is this constraint needed?

Reply:
It's not required by `PBftCrypto`, I suspect because we don't have an instance in scope for all kinds of keys?